### PR TITLE
Add panda3d yaml config

### PR DESCRIPTION
**Pull request by Maxwell175**
Thank your for contributing to Nuitka!

!! Please check that you select the **develop branch** (details see below link) !!

Before submitting a PR, please review the guidelines:
[Contributing Guidelines](https://github.com/kayhayen/Nuitka/blob/develop/CONTRIBUTING.md)

# What does this PR do?

This PR adds support of implicitly loaded panda3d dependencies.

# Why was it initiated? Any relevant Issues?

Panda3D has a number of implicitly imported dlls, which causes compiled executables to fail to run.

# PR Checklist

- [x] Correct base branch selected? Should be `develop` branch.
- [x] Enabled commit hook or executed `./bin/autoformat-nuitka-source`.
- [x] All tests still pass. Check the Developer Manual about `Running the Tests`.
      There are GitHub Actions tests that cover the most important
      things however, and you are welcome to rely on those, but they might not
      cover enough.
- [ ] Ideally new features or fixed regressions ought to be covered via new tests.
- [ ] Ideally new or changed features have documentation updates.

### CODE REVIEW

File: nuitka/plugins/standard/standard.nuitka-package.config.yml [hunk 1]
```diff
@@ -980,6 +980,25 @@
         - "osgeo._ogr"
         - "osgeo._osr"
 
+- module-name: "panda3d"
+  dlls:
+    - dest_path: "."
+      from_filenames:
+        prefixes:
+        - "libpandagl"
```
Comment (kayhayen) [inline nuitka/plugins/standard/standard.nuitka-package.config.yml]: Are these also for Windows? In that case a `when` clause might be missing.

Comment (Maxwell175) [reply nuitka/plugins/standard/standard.nuitka-package.config.yml]: The only one that is definitely windows only is libpandadx9. The rest are cross platform. (But can be disabled by people who do unofficial builds)

**Comment by kayhayen** [comment]
Can you try to rebase this one @Maxwell175 I think it's resolved and could well be in 1.1 then

**Comment by Maxwell175** [comment]
> Can you try to rebase this one @Maxwell175 I think it's resolved and could well be in 1.1 then

Done.

**Comment by kayhayen** [comment]
Ok, I think I can take it from here.

**Comment by kayhayen** [comment]
Manually merged

**Pull request closed without merging**